lte/gateway/c/core/oai/tasks/nas/esm/PdnConnectivity.c

```diff
@@ -550,9 +550,6 @@ proc_tid_t pdn_connectivity_delete(
         "\n",
         pdn_cid, ue_mm_context->mme_ue_s1ap_id);
   }
-  if (ue_mm_context->nb_active_pdn_contexts > 0) {
-    ue_mm_context->nb_active_pdn_contexts -= 1;
-  }
```

Comment:
rather than removing should not we simply say:
```
if ((ue_context_p->nb_active_pdn_contexts > 0) &&
      (ue_context_p->emm_context.esm_ctx.is_pdn_disconnect)) {
    ue_context_p->nb_active_pdn_contexts -= 1;
  }
```
as this is a counterpart to https://github.com/magma/magma/blob/c1f15ee8cf1ff8eb60cadcf7bfcc504e3806564d/lte/gateway/c/core/oai/tasks/mme_app/mme_app_bearer.c#L902 ?

Reply:
I missed that you actually removed the check `(!ue_context_p->emm_context.esm_ctx.is_pdn_disconnect)`